PythonVisualizations/SimpleQueue.py

```diff
@@ -76,40 +101,47 @@ def insertRear(self, val):
             self.onOffButtons()
             
             # update window
-            window.update()
-            
+            self.window.update()
+
+            self.list[self.rear] = drawable(val, Queue.colors[Queue.nextColor], cell, cell_val)
+            didInsert = True
         
         #if there's a space to insert into
         elif self.nItems != self.size:
 
             #deal with wraparound
             if self.rear == self.size-1:
                 self.rear = -1
+                # move rear arrow back to the beginning
+                self.moveItemsBy(self.rearArrow, (CELL_SIZE * self.size * -1, 0), steps=1)
```

Comment:
Moving items with `steps=1` will make the animation jump from one position to the destination.  The other visualizations make the pointer variables like rearArrow "slide" to their new position.  We should do the same for the Queue.